Source/WebCore/rendering/style/GridPositionsResolver.cpp

```diff
@@ -381,10 +381,10 @@ static void adjustGridPositionsFromStyle(const RenderBox& gridItem, GridTrackSiz
 
         if (initialPosition.isAuto()) {
             // Set initial position to span <line names - 1>
-            initialPosition.setSpanPosition(std::max(1, lineCount - 1), emptyString());
+            initialPosition.setSpanPosition(std::max(1, lineCount - 1), String());
```

Comment:
Changing to the String() looks fine. Was not super clear to me why this was set to an empty string in the first place from reading the past commits.


The grid spec was not super helpful either from a read through.

We could also do checks for `isEmpty` instead of `isNull`, but I am not sure we would gain much benefit.